libraries/botbuilder-core/tests/teams/test_teams_info.py

```diff
@@ -218,6 +218,24 @@ async def test_get_participant(self):
         handler = TeamsActivityHandler()
         await handler.on_turn(turn_context)
 
+    async def test_get_meeting_info(self):
+        adapter = SimpleAdapterWithCreateConversation()
+
+        activity = Activity(
+            type="message",
+            text="Test-get_meeting_info",
+            channel_id=Channels.ms_teams,
+            from_property=ChannelAccount(aad_object_id="participantId-1"),
+            channel_data={
+                "meeting": {"id": "meetingId-1"}
+            },
+            service_url="https://test.coffee",
+        )
+
+        turn_context = TurnContext(adapter, activity)
+        handler = TeamsActivityHandler()
+        await handler.on_turn(turn_context)
+
```

Comment:
Note: This test doesn't really do anything, as far as I can tell. Many of the other tests in here don't, either. I think I could add something like the `responses` package/module to intercept HTTP requests, but I didn't see that we do that anywhere else in the Python SDK. Let me know if you want this test changed in any way.

Reply:
You are correct.  It appears these tests are not complete.  For other repos like Java and C#, there is a TestTeamsActivityHandler that provides the actual calls and assertions.  See C# TeamsInfoTests.TestTeamsActivityHandler.

Reply:
It could possible be hairier.  Both Java and C# use mock ConnectorClients to mock the actual http calls.

Reply:
We're going to defer the unit tests in Python.  They haven't been functional to date.  Given the time, scope, and value, going to push the tests to another release.